code/session-1/GraphQL/Migrations/20240807140835_Initial.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace ConferencePlanner.GraphQL.Migrations
+{
```

Comment:
can you use the namespace declaration without the curly braces?

Reply:
This is a generated file (at least, initially). I'm not sure if it makes sense to instruct the reader to make this change.

See also: https://github.com/dotnet/efcore/issues/14545.